gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -426,18 +396,24 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
    if (where.empty())
       where = gROOT->GetWebDisplay().Data();
 
-   bool is_native = where.empty() || (where == "native"),
-        is_local = where == "local", // either cef or qt5
-        is_chrome = (where == "chrome") || (where == "chromium"),
-        is_firefox = (where == "firefox");
+   enum { kNative, kLocal, kChrome, kFirefox, kCEF, kQt5 } kind = kNative;
 
-#ifdef R__HAS_CEFWEB
+   if (where == "local")
+      kind = kLocal;
+   else if (where == "firefox")
+      kind = kFirefox;
+   else if ((where == "chrome") || (where == "chromium"))
+      kind = kChrome;
+   else if (where == "cef")
+      kind = kCEF;
+   else if (where == "qt5")
+      kind = kQt5;
```

Comment:
`else` error?

Reply:
Not error - it can be name of alternative web browser. 
I add more code and more comments